src/components/create-from-builder/create-from-builder.html

```diff
@@ -1,6 +1,6 @@
 <div class="order-service">
   <pf-wizard
-       hide-header="true"
+       title="{{$ctrl.imageStream.name}} {{$ctrl.istag.name}}"
```

Comment:
Can you check that this updates correctly when you pick different versions in the dropdown inside the wizard?

Reply:
Confirmed that it does.  :)